sdk/appconfiguration/azure-appconfiguration/README.md

```diff
@@ -172,86 +177,94 @@ config_setting = ConfigurationSetting(
 
+<!-- SNIPPET:hello_world_sample.get_config_setting -->
 ```python
 fetched_config_setting = client.get_configuration_setting(
-    key="MyKey", label="MyLabel"
+    key="MyKey"
 )
 ```
+<!-- END SNIPPET -->
 
 ### Delete a Configuration Setting
 
 Delete an existing Configuration Setting.
 
+<!-- SNIPPET:hello_world_advanced_sample.delete_config_setting -->
 ```python
-deleted_config_setting = client.delete_configuration_setting(
-    key="MyKey", label="MyLabel"
+client.delete_configuration_setting(
+    key="MyKey",
+    label="MyLabel",
 )
 ```
+<!-- END SNIPPET -->
 
 ### List Configuration Settings
 
 List all configuration settings filtered with label_filter and/or key_filter.
 
+<!-- SNIPPET:hello_world_advanced_sample.list_config_setting -->
 ```python
-
-filtered_listed = client.list_configuration_settings(
-    label_filter="My*", key_filter="My*"
-)
-for item in filtered_listed:
-    pass  # do something
-
+config_settings = client.list_configuration_settings(label_filter="MyLabel")
+for item in config_settings:
+    print_configuration_setting(item)
 ```
+<!-- END SNIPPET -->
 
 ### Async APIs
 
 Async client is supported.
 To use the async client library, import the AzureAppConfigurationClient from package azure.appconfiguration.aio instead of azure.appconfiguration
 
+<!-- SNIPPET:hello_world_sample_async.create_app_config_client -->
 ```python
-from azure.appconfiguration.aio import AzureAppConfigurationClient
+CONNECTION_STRING = get_connection_string()
 
-connection_str = "<connection_string>"
-async_client = AzureAppConfigurationClient.from_connection_string(connection_str)
+# Create app config client
+client = AzureAppConfigurationClient.from_connection_string(CONNECTION_STRING)
 ```
+<!-- END SNIPPET -->
 
 This async AzureAppConfigurationClient has the same method signatures as the sync ones except that they're async.
 For instance, to retrieve a Configuration Setting asynchronously, async_client can be used:
 
+<!-- SNIPPET:hello_world_sample_async.get_config_setting -->
 ```python
-fetched_config_setting = await async_client.get_configuration_setting(
-    key="MyKey", label="MyLabel"
+print("Get configuration setting")
+fetched_config_setting = await client.get_configuration_setting(
+    key="MyKey"
 )
+print("Fetched configuration setting:")
+print_configuration_setting(fetched_config_setting)
+print("")
 ```
+<!-- END SNIPPET -->
 
 To use list_configuration_settings, call it synchronously and iterate over the returned async iterator asynchronously
 
+<!-- SNIPPET:hello_world_advanced_sample_async.list_config_setting -->
 ```python
-
-filtered_listed = async_client.list_configuration_settings(
-    label_filter="My*", key_filter="My*"
-)
-async for item in filtered_listed:
-    pass  # do something
-
+print("List configuration settings")
+config_settings = client.list_configuration_settings(label_filter="MyLabel")
+async for item in config_settings:
+    print_configuration_setting(item)
 ```
+<!-- END SNIPPET -->
```

Comment:
We don't have code snippets for aync `set` and `delete`?

Reply:
It was not in the readme.

Reply:
Can we add them?

Reply:
The question is the formatting. :)

Let's open an issue to track it.